### PR TITLE
Acs

### PR DESCRIPTION
**Pull request by howew**

### CODE REVIEW

File: src/main/java/com/imsweb/algorithms/yostacspoverty/YostAcsPovertyOutputDto.java [hunk 1]
```diff
@@ -0,0 +1,294 @@
...
+    private String
+            _yostQuintile0610US, _yostQuintile0610State, _acsPctPov0610AllRaces, _acsPctPov0610White, _acsPctPov0610Black,
+            _yostQuintile1014US, _yostQuintile1014State, _acsPctPov1014AllRaces, _acsPctPov1014White, _acsPctPov1014Black,
+            _yostQuintile1418US, _yostQuintile1418State, _acsPctPov1418AllRaces, _acsPctPov1418White, _acsPctPov1418Black,
+            _acsPctPov0610AIAN, _acsPctPov0610AsianNHOPI, _acsPctPov0610OtherMulti, _acsPctPov0610WhiteNonHisp, _acsPctPov0610Hispanic,
+            _acsPctPov1014AIAN, _acsPctPov1014AsianNHOPI, _acsPctPov1014OtherMulti, _acsPctPov1014WhiteNonHisp, _acsPctPov1014Hispanic,
+            _acsPctPov1418AIAN, _acsPctPov1418AsianNHOPI, _acsPctPov1418OtherMulti, _acsPctPov1418WhiteNonHisp, _acsPctPov1418Hispanic;
+
+    public YostAcsPovertyOutputDto() {
+        _yostQuintile0610US = "";
```
Comment (depryf) [inline src/main/java/com/imsweb/algorithms/yostacspoverty/YostAcsPovertyOutputDto.java:16]: @howew the changes look good. The only thing I find weird is that you initialize all the output fields to a blank string. 

Unless you have a technical reason to do that, it's not good practice. Null is the default way in Java to indicate no value; I don't see any advantages in using an empty string to mean the same thing.

I don't believe any of the other algorithms do this.

Would you consider removing this constructor (or emptying it if you prefer)?

**Comment by howew** [comment]
Sure.  I just blanked it out.

**Comment by depryf** [comment]
@howew it looks like this branch was very out-of-date with master. The next time you start working on an issue please make sure to checkout master, update it, and then create your branch (or in this case, merge master into the existing branch). 

Two weeks ago, you worked on an issue to tweak the PRCDA and UIHO algorithms (pull request 95). Part of that issue was to change CountryData to initialize the data separately instead of as one.  These changes are are undoing those changes. Is that intended?

I am trying to merge master into this branch, but this is really complicated. I might need your help.

**Comment by howew** [comment]
Sorry about that.  It was not my intention to undo those changes.

**Comment by depryf** [comment]
OK, I am going to try to recover those, I will ask you if I am unable to figure it out.

**Comment by howew** [comment]
ok

**Pull request merged**